lib/pbench/test/unit/server/test_unpack_tarballs.py

```diff
@@ -240,13 +248,11 @@ def tickmock(id: str, ret_val: Any) -> Any:
 
         monkeypatch.setattr(os, "symlink", lambda path, dest: tickmock("symlink", None))
         monkeypatch.setattr(os, "unlink", lambda path: tickmock("unlink", None))
-        obj = UnpackTarballs(server_config, make_logger)
+        obj = UnpackTarballs(MockConfig, make_logger)
         obj.update_symlink(self.tar)
-        assert mocks_called == ["symlink", "symlink", "unlink"]
+        assert mocks_called == ["symlink", "unlink"]
```

Comment:
You might consider adding a second link (e.g., `"TO-INDEX, TO-COPY-SOS"`) to the `MockConfig` object to drive the loop in the CUT (which would return us to having two `"symlink"` entries here).

Reply:
Although ... not as an objection necessarily, but once this PR is submitted I'll be merging it into #2994 which intends to completely remove the concept of "state directories" and linked tarballs, so it seems that putting much extra work into "excessively detailed" testing here might not be the best use of time. 😆

Reply:
Fair enough.  Although, in the process here, Riya is becoming a Unit Test Maven!  😁